BlazarBase/src/main/java/com/hubspot/blazar/base/ModuleBuild.java

```diff
@@ -73,7 +77,8 @@ public ModuleBuild(@JsonProperty("id") Optional<Long> id,
                      @JsonProperty("endTimestamp") Optional<Long> endTimestamp,
                      @JsonProperty("taskId") Optional<String> taskId,
```

Comment:
In the UI I'm using the taskId field directly to generate a link to Singularity when debug mode is enabled. We need to update this class somehow so that I can obtain the correct singularity url.